projects/angular-auth-oidc-client/src/lib/validation/state-validation.service.ts

```diff
@@ -62,8 +64,8 @@ export class StateValidationService {
         }
 
         // its aud Claim Value MUST be the same as in the ID Token issued when the original authentication occurred,
-        if (decodedIdToken.aud !== newIdToken.aud) {
-            this.loggerService.logDebug(`aud do not match: ${decodedIdToken.aud} ${newIdToken.aud}`);
+        if (!this.equalityService.areRefreshEqual(decodedIdToken?.aud, newIdToken?.aud)) {
+            this.loggerService.logDebug(`aud do not match: '${decodedIdToken?.aud}' '${newIdToken.aud}'`);
```

Comment:
change 

this.loggerService.logDebug(`aud do not match: '${decodedIdToken?.aud}' 

to

this.loggerService.logDebug(`aud in new id_token is not valid: '${decodedIdToken?.aud}'